tests/pytest/test_multi.py

```diff
@@ -680,7 +679,8 @@ def testObjLenCommand(env):
     r.assertEqual(res, None)
 
     # Test missing path
-    r.expect('JSON.OBJLEN', 'doc1', '.nowhere').raiseError()
+    res = r.execute_command('JSON.OBJLEN', 'doc1', '.nowhere')
```

Comment:
```suggestion
    r.expect('JSON.OBJLEN', 'doc1', '.nowhere').equal(None)
```

Reply:
Keeping `execute_command` to handle cluster connection.